pkg/types/intoto/v0.0.2/intoto_v0_0_2_schema.json

```diff
@@ -0,0 +1,101 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://rekor.sigstore.dev/types/intoto/intoto_v0_0_2_schema.json",
+    "title": "intoto v0.0.2 Schema",
+    "description": "Schema for intoto object",
+    "type": "object",
+    "properties": {
+        "content": {
+            "type": "object",
+            "properties": {
+                "envelope": {
+                    "description": "dsse envelope",
+                    "type": "object",
+                    "properties": {
+                        "payload": {
+                            "description": "payload of the envelope",
+                            "type": "string",
+                            "writeOnly": true
+                        },
+                        "payloadType": {
+                            "description": "type describing the payload",
+                            "type": "string"
+                        },
+                        "signatures": {
+                            "description": "collection of all signatures of the envelope's payload",
+                            "type": "array",
+                            "minItems": 1,
+                            "items": {
+                                "description": "a signature of the envelope's payload along with the public key for the signature",
+                                "type": "object",
+                                "properties": {
+                                    "keyid": {
+                                        "description": "optional id of the key used to create the signature",
+                                        "type": "string"
+                                    },
+                                    "sig": {
+                                        "description": "signature of the payload",
+                                        "type": "string"
```

Comment:
this should probably have `"format": "byte"` added since it will be base64 encoded.

Reply:
Yes good catch. I also made the `payload` be required along with the `payloadType` and `signatures`
```
"required": ["payload", "payloadType", "signatures"],
```

Reply:
Actually this is already coming base64 encoded from the DSSE envelope so this will result in double encoding. Same for payload and signature
```
{
  "payload": "<Base64(SERIALIZED_BODY)>",
  "payloadType": "<PAYLOAD_TYPE>",
  "signatures": [{
    "keyid": "<KEYID>",
    "sig": "<Base64(SIGNATURE)>"
  }]
}
```

Reply:
I agree they'll already come base64 encoded, but the code should account for that. The schema should document the expectation that the values are base64 encoded.

Reply:
Would this not result double encoding? As a result we have to decode twice to get for example the payload and signature back?
```
re.IntotoObj.Content.Envelope.Payload = strfmt.Base64(env.Payload)
```

Reply:
Done.